### ISSUE TITLE
Subtask's view has dissapeared in 1.0.41

### ISSUE THREAD

**Issue by Jluisito** [kanboard/kanboard#3145]
### Actual behaviour
With the new dashboard in 1.0.41 version, the subtaks view is missing.
This view was one of the Kanboard's strengths, that allow to see all the subtasks assigned to me **from any project** gathered in a single list.
Now, with the new dashboard, I see all my task and subtaks BUT separated by projects. I can be member of 30 projects at the same time, so I don't want to review all these project all the time. 
I want a single list with my assigned task and a single list with my assigned subtask, not separated by project.

With the tasks I can solve it with the search "assignee:me". The result is the list of all my tasks of any project, that I can sort by several parameters.

With the subtasks I can't do that. I can search by "subtask:assignee:me", but the result is the list of tasks that contains subtask assigned to me. I can't see the list of subtask directly. I only can hover the mouse over the subtasks icon, but this action list all the subtasks of this task (completed or not, assigned to me or not...).

### Expected behaviour

There should be a view to see the list of all subtasks assigned to me. It could be the new dashboard view, but with the option (a toogle button?) of not separate tasks and subtasks by project.
Or modify the result of the "subtask:assignee:me" search to show the same list as now but with the subtasks assigned to me inside every task.

In short, with a few minor changes you can recover a really important Kanboard feature.

### Configuration
- Kanboard version: 1.0.41
- Database type and version: sqlite 3.6.20
- PHP version: 5.5.36
- OS: Linux 2.6.32-573.18.1.el6.x86_64
- Browser: Chrome 56.0.2924.87

**Comment by Chaosmeister** [comment]
This can be closed with version 1.0.43.
